tests/unit/ConfirmDialog.spec.ts

```diff
@@ -2,7 +2,7 @@ import Vue from 'vue'
 import Vuetify from 'vuetify'
 import Affix from 'vue-affix'
 import { mount } from '@vue/test-utils'
-import { ConfirmDialog } from '@/components/ConfirmDialog'
+import { ConfirmDialog } from '@/components/confirm-dialog'
```

Comment:
I'm confused as to how this works. As far as this unit test is concerned, ConfirmDialog is a local component in `@/components/ConfirmDialog`, isn't it?

(Same comment for ActionChip in ShareStructure.spec.ts.)

Reply:
The issue i'm having with our directory structure, is after we publish these packages and import them. The directory name is converted from `ActionChip` to `action-chip`.

![image](https://user-images.githubusercontent.com/53542131/111016402-a13b8c00-8362-11eb-9592-38431b7f2c35.png)

So when you have a component that utilizes another shared component, the import path is wrong in the component on the side of implementation. 

So i just renamed the directory to match the name it's converted too anyways. ie `action-chip` etc. I'm not sure of a way around this, but this conversion is an issue.

Reply:
So we can maybe go two ways here...
Update all the directory names to the format they become on the side of integration 
OR
figure out why/how these conversions happen and handle it.

Either way, it is a bit of tech debt.